src/backend/cpu/convolve.cpp

```diff
@@ -8,20 +8,31 @@
  ********************************************************/
 
 #include <Array.hpp>
+#include <arith.hpp>
+#include <blas.hpp>
+#include <cast.hpp>
 #include <common/defines.hpp>
 #include <convolve.hpp>
+#include <handle.hpp>
 #include <kernel/convolve.hpp>
 #include <platform.hpp>
+#include <reorder.hpp>
+#include <transpose.hpp>
+#include <unwrap.hpp>
+#include <wrap.hpp>
+#include <iostream>
```

Comment:
Please correct me if I'm wrong, but I don't think iostream is actually being used here. Please remove this include if so.